optimum/exporters/openvino/__main__.py

```diff
@@ -213,6 +218,9 @@ def main_export(
         model_name_or_path, subfolder=subfolder, revision=revision, cache_dir=cache_dir, token=token
     )
 
+    if framework == "pt":
+        import torch
+
```

Comment:
I would prefer if imports are mostly at the top of the file, conditioned by the package's availability.

Reply:
I can not agree, it will lead to import some packages that can be not required during model export (e.g. import tf, if you export pytorch model) and leads to some failure that will block model export however these packages actually do not required to export model

Reply:
yes for tensorflow it's understandable, but torch is a required dependency in optimum either ways, and used at multiple stages of the export.

Reply:
ok, moved